README.md

```diff
@@ -20,12 +20,29 @@ Two Slurm scripts are included in this repo.  If you want to run Rstudio to do d
 
 To run the pipeline as a non-interactive batch job, open `run_scmake.slurm` and modify parameters like job length (`--time`) and partition (`-p`) as needed for how long you expect the job to run (see [this issue](https://github.com/USGS-R/national-flow-observations/issues/4) for estimates). Submit the job with `sbatch --mail-user=$USER@usgs.gov run_scmake.slurm`.
 
-To be able to run the full pipeline, you will need to have AWS credentials setup on your user's home directory on Tallgrass. You need to have the file `/home/username/.aws/credentials`, which looks like this:
+To be able to run the full pipeline, you will need to have AWS credentials setup on your user's home directory on Tallgrass. This will only be possible for USGS users. You need to have the file `/home/username/.saml2aws`, which looks like this:
 
 ```
 [default]
-aws_access_key_id = ***
-aws_secret_access_key = ***
+   url                  = http://awsconsole.usgs.gov/
```

Comment:
Does this work? I have

```
url                     = https://fs.doi.gov
```

Reply:
What I have does work for me. I was confused by that `fs.doi.gov` because that it not the URL to AWS ...

Reply:
Neat. `fs.doi.gov` is just doing AD authentication and then establishing a token and sharing it between you and AWS. awsconsole.usgs.gov does the same thing, but I had thought that it would be doing it in a web-browser specific way which wouldn't work for AWS at the shell level. Good to know, thanks!

Reply:
Oh, likewise on the "thanks". I didn't understand what that URL was. Very cool

Reply:
Though, probably best to give people the most generic options for login since this saml2aws file might be used by them for other activities, so I will change it.